server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryErrorTest.java

```diff
@@ -73,27 +68,47 @@
 
 import com.google.common.collect.ImmutableList;
 
-import io.restassured.RestAssured;
-import io.restassured.builder.RequestSpecBuilder;
 import io.restassured.builder.ResponseSpecBuilder;
-import io.restassured.http.ContentType;
-import io.restassured.specification.RequestSpecification;
 import io.restassured.specification.ResponseSpecification;
-import net.javacrumbs.jsonunit.core.Option;
 
 public class RemoteDeliveryErrorTest {
+
     private static final Logger LOGGER = LoggerFactory.getLogger(RemoteDeliveryErrorTest.class);
 
     private static final String ANOTHER_DOMAIN = "other.com";
     private static final String FROM = "from@" + DEFAULT_DOMAIN;
+    private static final String RECIPIENT = "touser@" + ANOTHER_DOMAIN;
+    private static final String RECIPIENT1 = "touser1@" + ANOTHER_DOMAIN;
+    private static final String RECIPIENT2 = "touser2@" + ANOTHER_DOMAIN;
+
+    private static final MailAddress FROM_ADDRESS;
+    private static final MailAddress RECIPIENT_ADDRESS;
+    private static final MailAddress RECIPIENT1_ADDRESS;
+    private static final MailAddress RECIPIENT2_ADDRESS;
+
+    static {
+        try {
+            FROM_ADDRESS = new MailAddress(FROM);
+            RECIPIENT_ADDRESS = new MailAddress(RECIPIENT);
+            RECIPIENT1_ADDRESS = new MailAddress(RECIPIENT1);
+            RECIPIENT2_ADDRESS = new MailAddress(RECIPIENT2);
+        } catch (AddressException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static final Mail.Envelope FROM_RECIPIENT_ENVELOPE = new Mail.Envelope(FROM_ADDRESS,
```

Comment:
could we provide a vararg for the second argument to avoid this ugly construction of a List